contracts/test/integration/IntegrationTestPriceBoundPSMUSDC.t.sol

```diff
@@ -115,8 +116,19 @@ contract IntegrationTestPriceBoundPSMUSDCTest is DSTest {
 
     /// @notice PSM is set up correctly and redeem view function is working
     function testGetRedeemAmountOut() public {
-        uint256 amountVoltIn = 100e12;
-        assertEq(psm.getRedeemAmountOut(amountVoltIn), 101);
+        uint256 amountVoltIn = 100e18;
+
+        uint256 currentPegPrice = oracle.getCurrentOraclePrice().div(1e12);
+        uint256 fee = (amountVoltIn * psm.redeemFeeBasisPoints()) /
+            Constants.BASIS_POINTS_GRANULARITY;
+
+        uint256 amountOut = ((amountVoltIn * currentPegPrice).div(1e18)) - fee;
```

Comment:
there should be a comment here explaining all the things that are happening and why